docs/Data/Semiring/Boolean.md

```diff
@@ -0,0 +1,28 @@
+## Module Data.Semiring.Boolean
+
+#### `BooleanSemiring`
+
+``` purescript
+newtype BooleanSemiring a
+  = BooleanSemiring a
+```
+
+Provides the semiring for any boolean algebra.
```

Comment:
So if we have two, I think we could sensibly reuse `Conj` and `Disj` from `monoids`. I seem to recall a discussion about that earlier today. Is there a reason not to add `Semiring` to those `newtype`s?